alephclient/api.py

```diff
@@ -353,6 +353,32 @@ def _bulk_chunk(
                     return
                 backoff(ae, attempt)
 
+    def write_entity(self, collection_id: str, entity: Iterable, **kw):
+        """Create a single entity via the API, in the given
+        collection.
+
+        params
+        ------
+        collection_id: id of the collection to use. This will overwrite any
+        existing collection specified in the entity
+        entity: A dict object containing the values of the entity
+        """
+        if isinstance(entity, Dict):
+            entity["collection_id"] = collection_id
+
+        for attempt in count(1):
+            url = self._make_url("entities")
```

Comment:
If you add the entity ID in here, it will also do updates. Maybe that could be used if an ID is defined? (To be clear: users can specify random IDs for entities, only the suffix is overwritten).

Reply:
Hmmm. From the memorious side, at the moment, it just generates a uuid. But thinking about it something repeatable might be better...